hilo.rb

```diff
@@ -1,26 +1,28 @@
+# Devtrain: Ruby Hi-Lo
+
 # predefined number of guesses allowed
 guesses = 6
```

Comment:
So it'd be better if this was in a class as it encapsulates all this logic so if you needed to add this to a larger thing(like a games platform) it would be much easier to use elsewhere.

like this
```
class HiLoGame
  def initialize
    @guesses = 6
    @number = ...
  end

  def play
    ..your code here
  end
end

HiLoGame.new.play
```